login.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+---
+<section class="main-part">
+    <div class="container">
+        <div class="mb-3"></div>
+        <div class="form-floating mb-3">
+            <input type="email" class="form-control" id="email-input" aria-label="Email" aria-describedby="email-label" placeholder="abc@xyz.com">
+            <label id="email-label" for="email-input">Email</label>
+        </div>
+        <div class="form-floating mb-3">
+            <input type="password" class="form-control" id="password-input" aria-label="Password" aria-describedby="password-label" placeholder="Password">
+            <label id="password-label" for="password-input">Password</label>
+        </div>
+        <div class="form-floating mb-3">
+            <input type="submit" id="submit-button" placeholder="Submit" value="Submit">
+        </div>
+    </div>
+</section>
+<script type="module">
+    import {trySignIn} from './assets/js/authentication.js'
+    const submitButton = document.getElementById("submit-button")
+    submitButton.onclick = trySignIn
```

Comment:
[Not recommended](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#inline_event_handlers_%E2%80%94_dont_use_these) and also doesn't work well with functions exported from modules because they aren't in the global scope